event_stage_cancelled/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,2 @@
+This module adds the posibility of flaggin an event stage as cancelled so we can hook
```

Comment:
I would put here all the speech about that this was present in v13 and so on, and also mention that the cancellation leads to registrations cancelling as well.

Reply:
Doesn't it fit better in the *Context* section?